packages/create-bison-app/template/cypress/plugins/index.ts

```diff
@@ -48,14 +42,15 @@ export const setupNodeEvents: Cypress.ConfigOptions['setupNodeEvents'] = (on, _c
 
       return Factory.create(attrs);
     },
-    login: async (attrs: Pick<User, 'email' | 'password'>): Promise<LoginTaskObject> => {
-      const { email, password } = attrs;
-      const variables = { email, password };
-
-      const response = await trpcRequest().user.login(variables);
-      const { token } = response;
-      cookies().set(LOGIN_TOKEN_KEY, token);
-      return { token };
+    mockValidSession: ({ user }: { user: NextAuthSession['user'] }): NextAuthSession => {
+      const isAdmin = user.roles.includes(Role.ADMIN);
+      return {
+        user,
+        isAdmin,
+        expires: new Date(Date.now() + 2 * 86400).toISOString(),
```

Comment:
Nit: I always prefer to calculate out my durations to make it clear what the duration is.

Also, the addition appears to be in seconds,  but `Date.now()` returns milliseconds, so this should be altered anyway.

```suggestion
        expires: new Date(Date.now() + 2 * 24 * 60 * 60 * 1000).toISOString(),
```